components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/util/ActionDTOBuilder.java

```diff
@@ -134,6 +138,17 @@ public EndpointConfig getEndpoint() {
         return this.endpoint;
     }
 
+    public ActionDTOBuilder rule(ActionRule rule) {
+
+        this.rule = rule;
+        return this;
+    }
+
+    public ActionRule getActionRule() {
```

Comment:
```suggestion
    public ActionRule getRule() {
```
Do we need to repeat the Action in the method name again ?

Reply:
yes. Because from ActionRule you can get the real Rule. So it should not be something like getRule().getRule()